aten/src/ATen/native/cuda/CuFFTPlanCache.h

```diff
@@ -262,11 +319,16 @@ class CuFFTConfig {
             inembed.data(), base_istride, idist, itype,
             onembed.data(), base_ostride, odist, otype,
             batch, &ws_size_t, exec_type));
-    }
+#endif
+      }
     ws_size = static_cast<int64_t>(ws_size_t);
   }
 
+#ifdef __HIP_PLATFORM_HCC__
+  cufftHandle &plan() const { return *plan_ptr.get(); }
```

Comment:
Why remove the constness here? The signatures for hipfftCreate & cufftCreate are identical.